CONTRIBUTING.md.bak

```diff
@@ -0,0 +1,655 @@
+# Galaxy Community Hub [![Build Status](https://jenkins.galaxyproject.org/buildStatus/icon?job=galaxy-hub)](https://jenkins.galaxyproject.org/job/galaxy-hub/)
```

Comment:
You probably do not want to commit this backup file?

Reply:
Probably not.  Sorry, first time I used aspell, and I was not paying attention.  I'll drop them.